Sources/Alloy/Core/Extensions/Metal/MTLDevice+Features.swift

```diff
@@ -15,6 +16,24 @@ public extension MTLDevice {
             #elseif os(macOS)
             return self.supportsFeatureSet(.macOS_GPUFamily1_v3)
             #endif
+            
+        case let .readWriteTexture(pixelFormat):
```

Comment:
I don't think that read_write texture is only pixel format dependent. It is surely also depends on GPU family

Reply:
yup, we get readWriteTextureSupport from current device and check if it supports this feature

Reply:
oh sorry I got it